File: src/main/java/no/nav/melosys/eessi/models/sed/nav/EndringA008Bruker.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

@JsonInclude(JsonInclude.Include.NON_NULL)
@Data
public class EndringA008Bruker {

        private String fornavn;

        private String etternavn;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/EndringA008Bruker.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class EndringA008Bruker (
    var fornavn: String? = null,
    var etternavn: String? = null
)

